src/test/java/example/WekaTestPlayground.java

```diff
@@ -0,0 +1,79 @@
+package example;
+
+import org.junit.Test;
+import weka.classifiers.bayes.NaiveBayes;
+import weka.classifiers.functions.LinearRegression;
+import weka.core.Attribute;
+import weka.core.FastVector;
+import weka.core.Instance;
+import weka.core.Instances;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.junit.Assert.assertTrue;
+
+/**
+ * Created by Timotej on 29-Oct-15.
+ */
+public class WekaTestPlayground {
+    Attribute x = new Attribute("x");
```

Comment:
Would you mind documenting:
* What the file does
* How some of the Objects interact (e.g. what are the purposes of Instance and Attribute)

If you do, we could use this as a primer for people moving onto working with weka in the furture